test-cases/gutenberg/writing-flow/splitting-merging.md

```diff
@@ -40,6 +40,9 @@ Start from an empty post.
 - Delete all those words until the blocks merge.
 - Check that the blocks were merged.
 
+**Known Issues**
+- Does not merge on Android after deleting either styled text or a selection containing multiple charaacters (instead of single characters): [#1873](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1873#issuecomment-608070758).
+
```

Comment:
From https://github.com/wordpress-mobile/WordPress-Android/pull/11572#issuecomment-607352657 - TC004 also has the same issue. Blocks won't merge after deleting all text.

Maybe prefixing with [Android] is easier to spot platform specific issues?

Reply:
Good idea! 👍 Made the change.